update to 1.8.0

katietz opened the pull request.

skupr-anaconda commented:
As cph dropped python 2.7 support https://github.com/conda/conda-package-handling/blob/1.8.0/CHANGELOG.rst#deprecations we can remove `six` and `futures` from `req/run`

And it would be nice to add dev_url: http`s://github.com/conda/conda-package-handling`

katietz commented:
I added the devurl.  I also removed 'futures' dependency.  Just checking if 'six' is for sure not used, as by looking through the source it seems to be used in context of mock. I am just run the testing for it

jezdez commented:
Thank you all!
